modules/top-level/output.nix

```diff
@@ -121,7 +216,9 @@ with lib;
       init = helpers.writeLua "init.lua" customRC;
 
       extraWrapperArgs = builtins.concatStringsSep " " (
-        (optional (config.extraPackages != [ ]) ''--prefix PATH : "${makeBinPath config.extraPackages}"'')
+        (optional (
+          config.extraPackages != [ ]
+        ) ''--prefix PATH : "${lib.makeBinPath config.extraPackages}"'')
```

Comment:
This is just removing `with lib`? Could that be done in a dedicated PR? It makes it more difficult to review this one

Reply:
It's in a separate first commit, changes are minimal. I did it to make language server smarter about undeclared variables and such. But yes, I can create a branch from the first commit and make a PR for it, but it's kind of strange to do it only for one file.

Reply:
If it's needed elsewhere in this PR and it's a small change in its own commit then that's fine.

Reply:
Yeah at some point I'd like to remove all `with lib;`